components/brave_wallet_ui/components/extension/transaction-list-item/index.tsx

```diff
@@ -0,0 +1,206 @@
+import * as React from 'react'
+import * as EthereumBlockies from 'ethereum-blockies'
+
+import { getLocale } from '../../../../common/locale'
+import {
+  BraveWallet,
+  WalletAccountType,
+  DefaultCurrencies
+} from '../../../constants/types'
+
+// Utils
+import { toProperCase } from '../../../utils/string-utils'
+import { getTransactionStatusString } from '../../../utils/tx-utils'
+import { mojoTimeDeltaToJSDate, formatDateAsRelative } from '../../../utils/datetime-utils'
+import { formatFiatAmountWithCommasAndDecimals } from '../../../utils/format-prices'
+
+// Hooks
+import { useTransactionParser } from '../../../common/hooks'
+import { SwapExchangeProxy } from '../../../common/hooks/address-labels'
+
+// Styled Components
+import {
+  StatusBubble,
+  DetailTextDarkBold,
+  DetailTextDark,
+  DetailTextLight
+} from '../shared-panel-styles'
+
+import {
+  ArrowIcon,
+  BalanceColumn,
+  DetailColumn,
+  DetailRow,
+  FromCircle,
+  StatusRow,
+  StyledWrapper,
+  ToCircle,
+  TransactionDetailRow
+} from './style'
+
+export interface Props {
+  selectedNetwork: BraveWallet.EthereumChain
+  transaction: BraveWallet.TransactionInfo
+  account: WalletAccountType | undefined
```

Comment:
I guess same as this:

```suggestion
  account?: WalletAccountType
```

Reply:
Done!